### DISCUSSION TITLE
Make divider a customized component.

### DISCUSSION THREAD

**Discussion by Sepulven** [saadeghi/daisyui#2489] [Ideas / feature requests]
While making some layouts for my web page, I've noticed the lack of customization of divider component class.

 As you can see here: 
https://daisyui.com/components/divider/

Suggestion:

- Different colors (primary, neutral, base, secondary...);
- Width sizes;
- Line's style (dotted, wavy, dashed);
- Position of the content (start, middle, end);

I hope I've given a good suggestion, it may be a component that is not so used but for me it was important.
Feel free to give your feedback on my idea!

**Comment by saadeghi** [comment]
Thanks. 
Color modifier classes and position modifier classes are already done in the upcoming v4 and will be released really soon.  
I will also consider your other suggestions for the future updates 🙌
